process/graduation_criteria.md

```diff
@@ -39,7 +40,5 @@ To graduate from sandbox or incubating status, or for a new project to join as a
  * Have committers from at least two organizations.
  * Have achieved and maintained a Core Infrastructure Initiative [Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
  * Have completed an independent and third party security audit with results published of similar scope and quality as the following example (including critical vulnerabilities addressed): https://github.com/envoyproxy/envoy#security-audit and all critical vulnerabilities need to be addressed before graduation.
- * Explicitly define a project governance and committer process. The committer process should cover the full committer lifecycle including onboarding and offboarding or emeritus criteria. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
```

Comment:
OK, we don't want to just remove this without replacing it.  By the time a project has gotten to graduation, they should have evolved (and, for that matter, used) their governance.  That is, the expectations for transparency, accessibility, and completeness for a graduated project should be higher than they were for sandbox, and we should call that out as a requirement.

Draft text:

Have complete and detailed governance, both in documentation and in practice, that address all aspects of managing the project, including contributor roles, maintainer selection and departure, handling code of conduct and security incidents, and management of any subprojects.  This governance should be open to all contributors and non-prejudicial towards any contributor affiliation.

Reply:
Also, question: if we're requiring governance at sandbox entry, what are we requiring at Incubation?